### DISCUSSION TITLE
Editor

### DISCUSSION THREAD

**Discussion by gamecoder-nz** [bevyengine/bevy#7783] [Ideas]
Is anyone working on a full scene and project editor?

**Comment by alice-i-cecile** [comment]
Yep, we're slowly progressing on external prototypes, requirements gathering and required engine features. See https://github.com/bevyengine/bevy/discussions/7100 for a great discussion of the core requirements and architecture.

**Comment by BlackPhlox** [comment]
One of the external prototypes is [Potoo](https://github.com/BlackPhlox/potoo), which is more of a project editor prototype/experimental thing and is in a very early stage.
